src/electionguard/constants.py

```diff
@@ -0,0 +1,81 @@
+"""Creating and managing mathematic constants for the election."""
+from os import getenv
+
+from dataclasses import dataclass
+from enum import Enum
+
+
+@dataclass
+class ElectionConstants:
+    """The constants for mathematical functions during the election."""
+
+    large_prime: int
+    """large prime or p"""
+
+    small_prime: int
+    """small prime or q"""
+
+    cofactor: int  # (p - 1) / q
+    """cofactor or r"""
+
+    generator: int
+    """generator or g"""  # 2^r mod p
+
+
+def create_constants(
+    large_prime: int, small_prime: int, cofactor: int, generator: int
+) -> ElectionConstants:
+    """Create constants for election."""
+    return ElectionConstants(
+        large_prime,
+        small_prime,
+        cofactor,
+        generator,
+    )
+
+
+# pylint: disable=line-too-long
+STANDARD_CONSTANTS = create_constants(
```

Comment:
how do you feel about adding a comment here that includes a link to the spec where primes are specified?

Reply:
Added this in markdown